test/client/auth.hpp

```diff
@@ -120,6 +121,8 @@ class Client {
     /// This functions is thread-safe.
     void refresh(std::string refresh_token, std::function<RefreshHandler>);
 
+    static std::string make_http_host(bool auth_ssl, const std::string& auth_address, const port_type& auth_port);
```

Comment:
It was trying to fix some of the errors in the test/client application - since we're not even building this right now and it is going to take some work to get it current, I reverted all the changes not related to the logger.